src/index.js

```diff
@@ -120,10 +120,9 @@ export default class Spinner extends React.Component {
   _renderSpinner() {
     const { visible } = this.state;
 
-    if (!visible)
-      return (
-        <View />
-      );
+    if (!visible) {
+      return null;
```

Comment:
Strictly speaking, it returns `undefined`, but React will react to either value in the way that we want. So, sure thing. Do you have a style preference about braces/newlines for single-line if statements?